test/toolchains/BUILD.bazel

```diff
@@ -0,0 +1,77 @@
+load("//scala:scala_toolchain.bzl", "scala_toolchain")
+
+scala_toolchain(
+    name = "ast_plus_one_deps_unused_deps_warn_impl",
+    dependency_mode = "plus-one",
+    dependency_tracking_method = "ast",
+    unused_dependency_checker_mode = "warn",
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "ast_plus_one_deps_unused_deps_warn",
+    toolchain = "ast_plus_one_deps_unused_deps_warn_impl",
+    toolchain_type = "@io_bazel_rules_scala//scala:toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+scala_toolchain(
+    name = "ast_plus_one_deps_unused_deps_error_impl",
+    dependency_mode = "plus-one",
+    dependency_tracking_method = "ast",
+    unused_dependency_checker_mode = "error",
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "ast_plus_one_deps_unused_deps_error",
+    toolchain = "ast_plus_one_deps_unused_deps_error_impl",
+    toolchain_type = "@io_bazel_rules_scala//scala:toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+scala_toolchain(
+    name = "ast_plus_one_deps_strict_deps_warn_impl",
+    dependency_mode = "plus-one",
+    dependency_tracking_method = "ast",
+    strict_deps_mode = "warn",
+    visibility = ["//visibility:public"],
+)
+
+toolchain(
+    name = "ast_plus_one_deps_strict_deps_warn",
```

Comment:
Also this isn't used

Reply:
I think this is used in test_deps.sh on the last line of test_plus_one_ast_analyzer_strict_deps